.env.development

```
@@ -1 +1 @@
-BACKEND="http://fakedomain.com:5000"
\ No newline at end of file
+BACKEND="http://localhost:5000"
```

Jo das brauchen wir auf fakedomain ^^
```suggestion
BACKEND="http://fakedomain.com:5000"
```